test/tools/format_tool_test.dart

```diff
@@ -172,8 +208,13 @@ void main() {
             path: 'test/tools/fixtures/format/has_dart_style');
         expect(execution.exitCode, isNull);
         expect(execution.process.executable, 'pub');
-        expect(execution.process.args,
-            orderedEquals(['run', 'dart_style:format', '.']));
+        expect(
+            execution.process.args,
+            orderedEquals([
+              'run',
+              'dart_style:format',
+              'test/tools/fixtures/format/has_dart_style'
```

Comment:
In this case, we want the returned path to be relative to the path we passed to `buildExecution`. So I would expect this to still be `.` and not the full path to the test fixture